src/api/webhooks.js

```diff
@@ -96,8 +87,6 @@ export class WebhooksApi extends BaseApi {
      */
     delete(webhookId) {
         const url = this.getFullURL('webhooks', webhookId);
-        return this.validateAuthHeaders().then((authHeaders) => {
-            return http('DELETE', url, undefined, authHeaders);
-        });
+        return this.request('DELETE', url);
```

Comment:
How come in some instances you pass empty object for the third param, but here it's ok to leave it out?